raytracing/matrixgroup.py

```diff
@@ -92,14 +92,88 @@ def append(self, matrix):
         self.backVertex = transferMatrix.backVertex
 
     def __len__(self):
+        """
+        Returns the number of matrices in the group. Allows the use of len(MatrixGroup).
+        """
         return len(self.elements)
 
-    def __getitem__(self, item):
-        if isinstance(item, slice):  # If we get a slice, return a matrix group
-            return MatrixGroup(self.elements[item])
-        return self.elements[item]
+    def __getitem__(self, key):
+        """ This function is used to obtain a single element.
```

Comment:
This used to say "substitute" an element.  I corrected it, is it only for getting an element?

Reply:
Yes it is only for getting, it was a copy paster error